examples/seismic/tti/operators.py

```diff
@@ -105,13 +139,25 @@ def Gxx_centered_2d(field, costheta, sintheta, space_order):
     As the Laplacian is rotation invariant, it is computed as the conventional
     Laplacian minus the second order rotated second order derivative in the direction z
     Gxx = field.laplace - Gzz
-    :param field: symbolic data whose derivative we are computing
-    :param costheta: cosine of the tilt angle
-    :param sintheta:  sine of the tilt angle
-    :param cosphi: cosine of the azymuth angle
-    :param sinphi: sine of the azymuth angle
-    :param space_order: discretization order
-    :return: Sum of the 3D rotated second order derivative in the direction x
+
+    Parameters
+    ----------
+    field : FIX
+        Symbolic data whose derivative we are computing.
```

Comment:
this is not "Symbolic data ..." -- maybe something simpler, along the lines of "Input symbol." ?